package.json

```diff
@@ -41,7 +41,7 @@
       "url": "https://github.com/paullouisageneau"
     }
   ],
-  "license": "LGPL",
+  "license": "MPL 2.0",
```

Comment:
Possibly just a nitpick, but I think this should be the SPDX license identifier, so `"MPL-2.0"`.

I don't know if it's important for npm.